Adapter.C

```diff
@@ -110,6 +110,13 @@ bool preciceAdapter::Adapter::configFileRead()
                     }
                     DEBUG(adapterInfo("    connectivity : " + std::to_string(interfaceConfig.meshConnectivity)));
 
+                    // When restarting FSI simulations, we may need to account for previous displacement.
+                    // We do this by resetting the displacement when defining the interface.
+                    // Since this is a feature that may not work as expected, depending on the implementation of the
+                    // structure solver used, we allow the user to enable this feature.
```

Comment:
I think this better reflect what you want to say:

```suggestion
                    // structure solver used, we make this feature opt-in.
```